ADOPTERS.md

```diff
@@ -13,6 +13,8 @@ See list of adopter types at the bottom of this page.
 | [Edgeless Systems](https://www.edgeless.systems/)                 | [Contrast](https://github.com/edgelesssys/contrast)           | Beta                     | Service Provider / Consultancy   | Contrast runs confidential container deployments on Kubernetes at scale.                                |
 | [IBM](https://www.ibm.com/z)                                    | [IBM LinuxONE](https://www.ibm.com/linuxone)                  | Beta                     | Service Provider                 | Confidential Containers with Red Hat OpenShift Container Platform and IBM® Secure Execution for Linux (see [details](https://www.ibm.com/blog/confidential-containers-with-red-hat-openshift-container-platform-and-ibm-secure-execution-for-linux/))  |
 |NanhuLab|Trusted Big Data Sharing System |Beta |Service Provider |The system uses confidential containers to ensure that data users can utilize the data without being able to view the raw data.(No official website yet. For details: yzc@nanhulab.ac.cn) |
+| [Red Hat](https://www.redhat.com/en) | [OpenShift confidential containers](https://www.redhat.com/en/blog/learn-about-confidential-containers) | Beta | Service Provider | Confidential Containers are available from [OpenShift sandboxed containers release version 1.7.0](https://docs.redhat.com/en/documentation/openshift_sandboxed_containers/1.7/) as a tech preview on Azure cloud for both Intel TDX and AMD SEV-SNP. The tech preview also includes support for confidential containers on IBM Z and LinuxONE using Secure Execution for Linux (IBM SEL). Future releases will support bare metal deployments and additional public clouds. |
```

Comment:
Is `IBM SEL` the correct term?

Reply:
The text is a copy paste from our landing page: https://www.redhat.com/en/blog/learn-about-confidential-containers

So I believe it is

Reply:
I've never heard the term `SEL` for it. "IBM Secure Execution for Linux" is the correct name for it and I'm not sure there is an approved abbreviation, so I'd probably just suggest removing that?

Reply:
Our Product Manager has stated that SEL is being introduced as an approved abbreviation, so we can leave that as is. Thanks and sorry for any confusion.